transplant/transplant_master.py

```diff
@@ -523,6 +525,11 @@ def __init__(self, executable='matlab', arguments=tuple(), msgformat='msgpack',
                 arguments += '-minimize',
         if not jvm and '-nojvm' not in arguments:
             arguments += '-nojvm',
+        paths = (os.path.dirname(__file__))
+        if addpath is not None:
+            for path in addpath:
+                tempath = str('\',\'' + path)
+                paths += tempath
```

Comment:
The logic of inserting "single" quotation signs here is confusing.

How about replacing the `addpath('{}')` in line 546 with `addpath({})`, and building a properly-quoted list of strings here? I think this would make the whole thing easier to read. Maybe along the lines of

    paths = [os.path.dirname(__file__)] + addpath
    paths_string = ','.join("'{}'".format(p) for p in paths)

Reply:
Indeed.

Reply:
I agree.